src/connector.js

```diff
@@ -461,17 +475,13 @@ function initEngine() {
 					open: '<strong>',
 					close: '</strong>',
 				},
-				correctionDelimiters: {
-					open: '<em>',
-					close: '</em>',
-				},
 			},
 		}
 	} );
 
 	resultListController = buildResultList( headlessEngine, {
 		options: {
-			fieldsToInclude: [ "author", "date", "language", "urihash", "objecttype", "collection", "source", "permanentid", "displaynavlabel" ]
+			fieldsToInclude: [ "author", "date", "language", "urihash", "objecttype", "collection", "source", "permanentid", "displaynavlabel", "hostname" ]
```

Comment:
Can you please document this change in the description of your PR? Or open a separate PR with this change, since it is most likely unrelated to QS.